loopchain/blockchain/blockchain.py

```diff
@@ -98,6 +101,35 @@ def __init__(self, channel_name=None, store_id=None, block_manager=None):
         for tx_version, tx_hash_version in channel_option.get("hash_versions", {}).items():
             self.__tx_versioner.hash_generator_versions[tx_version] = tx_hash_version
 
+    @property
+    def leader_made_block_count(self):
+        if self.__last_block:
+            return self.__last_leader_made_block_count[self.__last_block.header.peer_id]
+        return -1
+
+    @property
+    def my_made_block_count(self):
+        return self.__last_leader_made_block_count[ChannelProperty().peer_address]
+
+    def _up_leader_made_block_count(self):
+        self.__last_leader_made_block_count[self.__last_block.header.peer_id] += 1
+
+    def reset_leader_made_block_count(self):
+        self.__last_leader_made_block_count.clear()
+
+    def get_next_leader(self):
```

Comment:
I think the return value is ExternalAddress (I'm not sure), but PeerManager.peer_id set its value as str.

I suggest below. but setting return type as ExternalAddress could breach style convention on IDE.
```suggestion
    def get_next_leader(self) -> ExternalAddress:
```